src/ray/gcs/gcs_client/test/service_based_gcs_client_test.cc

```diff
@@ -619,6 +619,22 @@ TEST_F(ServiceBasedGcsGcsClientTest, TestTaskInfo) {
   ASSERT_TRUE(AttemptTaskReconstruction(task_reconstruction_data));
 }
 
```

Comment:
Could you add some tests to make sure the sequencer work as exepcted?

Reply:
I have add sequencer_test.cc to make sure the sequencer work as exepcted.